File: add/codesnippet/CPP/e-system.configuration.i_3_1.cpp
public:
   MyInstallerClass()
   {
      // Attach the 'Committed' event.
      this->Committed += gcnew InstallEventHandler( this, &MyInstallerClass::MyInstaller_Committed );
   }

private:
   // Event handler for 'Committed' event.
   void MyInstaller_Committed( Object^ sender, InstallEventArgs^ e )
   {
      Console::WriteLine( "Committed Event occured." );
   }

File: add/codesnippet/CSharp/e-system.configuration.i_3_1.cs
   public MyInstallerClass() :base()
   {
      // Attach the 'Committed' event.
      this.Committed += new InstallEventHandler(MyInstaller_Committed);
   }

   // Event handler for 'Committed' event.
   private void MyInstaller_Committed(object sender, InstallEventArgs e)
   {
      Console.WriteLine("Committed Event occured.");
   }